integration_test/gce/gce_testing.go

```diff
@@ -451,29 +465,45 @@ func firstTrace(it *trace.TraceIterator) (*cloudtrace.Trace, error) {
 	return trace, nil
 }
 
-// WaitForMetric looks for the given metric in the backend and returns it if it
+// WaitForMetric uses WaitForMetricSeries to return a single metric or appropriate error
```

Comment:
Function headers should be self-contained IMO. I'd preserve the existing header for this function, and then the header for `WaitForMetricSeries` can pretty much be a duplicate of it, with the wording slightly changed to indicate that it returns a slice instead of a single time series.